elements/lisk-transactions/src/utils/verify.ts

```diff
@@ -98,8 +98,20 @@ export const validateKeysSignatures = (
 	transactionBytes: Buffer,
 ) => {
 	const errors = [];
+
 	// tslint:disable-next-line: prefer-for-of no-let
 	for (let i = 0; i < keys.length; i += 1) {
+		if (signatures[i].length === 0) {
+			errors.push(
+				new TransactionError(
+					'All mandatory keys have to sign the transaction.',
```

Comment:
```suggestion
					'Invalid signatures format. signatures should not include empty string.',
```

Reply:
Also, since it's array, we can `continue` instead of `break`

Reply:
but as this are mandatory it doesn't make sense to continue as all need to be present.